go/test/endtoend/onlineddl/vtgate_util.go

```diff
@@ -303,6 +316,7 @@ func WaitForThrottledTimestamp(t *testing.T, vtParams *mysql.ConnParams, uuid st
 		}
 		time.Sleep(1 * time.Second)
 	}
-	t.Error("timeout waiting for last_throttled_timestamp to have nonempty value")
+	require.Fail(t, fmt.Sprintf("failed to get a last_throttled_timestamp value for migration %s before hitting the timeout of %v",
+		uuid, timeout))
```

Comment:
Cool. For some reason I was thinking that didn't work as expected. I'll double check and then use this assuming it works.

Reply:
Going back to what I had. Now it prints:
```
Error:      	migration %s did not reach an expected status (%v) before hitting the timeout of %v, last known status: %v
```

Reply:
Ahhh, sorry for misleading -- it's `Failf()` that should be used.

Reply:
Surprisingly, `require.Failf()` works the same way in this regard. I know that because we were using it in VDiff2 tests and I had to use `fmt.Sprintf()`.